packages/mobile/src/send/SendConfirmation.tsx

```diff
@@ -221,65 +287,151 @@ export class SendConfirmation extends React.Component<Props, State> {
+            />
+          )}
+          <LineItemRow
+            title={t('securityFee')}
+            titleIcon={<FeeIcon />}
+            amount={
+              securityFeeAmount && (
+                <CurrencyDisplay amount={securityFeeAmount} formatType={FormatType.Fee} />
+              )
+            }
+            isLoading={asyncFee.loading}
+            hasError={!!asyncFee.error}
+          />
+          {/* <HorizontalLine /> */}
+          <TotalLineItem amount={totalAmount} />
+        </View>
+      )
     }
 
     return (
       <SafeAreaView style={styles.container}>
         <DisconnectBanner />
         <ReviewFrame
-          HeaderComponent={this.renderHeader}
-          FooterComponent={this.renderFooter}
+          FooterComponent={renderFeeContainer}
           confirmButton={primaryBtnInfo}
-          modifyButton={secondaryBtnInfo}
           shouldReset={this.state.buttonReset}
+          isSending={this.props.isSending}
         >
-          <TransferReviewCard
-            recipient={recipient}
-            address={recipientAddress || ''}
-            e164PhoneNumber={recipient.e164PhoneNumber}
-            comment={reason}
-            value={amount}
-            currency={CURRENCY_ENUM.DOLLAR} // User can only send in Dollars
-            fee={fee}
-            isLoadingFee={asyncFee.loading}
-            feeError={asyncFee.error}
-            type={type}
-            validatedRecipientAddress={validatedRecipientAddress}
-            onEditAddressClick={this.onEditAddressClick}
-          />
+          <View style={styles.transferContainer}>
+            {isInvite && <Text style={styles.inviteText}>{t('inviteMoneyEscrow')}</Text>}
+            <View style={styles.headerContainer}>
+              <ContactCircle
+                thumbnailPath={getRecipientThumbnail(recipient)}
+                address={recipientAddress || ''}
+              />
+              <View style={styles.recipientInfoContainer}>
+                <Text style={styles.headerText}>Sending</Text>
+                <Text style={styles.displayName}>
+                  {getDisplayName({ recipient, recipientAddress, t })}
+                </Text>
+                {validatedRecipientAddress && (
+                  <View style={styles.editContainer}>
+                    <Text style={styles.address}>{`${validatedRecipientAddress.slice(
+                      0,
+                      6
+                    )}...${validatedRecipientAddress.slice(-4)}`}</Text>
+                    <TextButton
+                      style={styles.editButton}
+                      testID={'accountEditButton'}
+                      onPress={this.onEditAddressClick}
+                    >
+                      {t('edit')}
+                    </TextButton>
+                  </View>
+                )}
+              </View>
+            </View>
+            <CurrencyDisplay
+              type={DisplayType.Big}
```

Comment:
Use `Default` instead and adjust the font you want using `style`.

Reply:
Nice job making this so extensible. Makes it really easy to use!